go.mod

```diff
@@ -16,11 +16,11 @@ require (
 	go.uber.org/zap v1.27.0
 	google.golang.org/grpc v1.72.0
 	k8s.io/apimachinery v0.33.1
-	k8s.io/client-go v0.32.4
-	k8s.io/component-base v0.32.4
+	k8s.io/client-go v0.32.5
+	k8s.io/component-base v0.32.5
 	sigs.k8s.io/controller-runtime v0.20.4
 	sigs.k8s.io/gateway-api v1.3.0
-	sigs.k8s.io/gateway-api-inference-extension v0.0.0-20250515212313-6e8a2effa41c
+	sigs.k8s.io/gateway-api-inference-extension v0.0.0-20250521193836-a5bf0acd13cc
```

Comment:
q: when is upstream expected to cut and tag a release? I thought v0.4 was just around the corner...

Reply:
yes. I thought as well. 
the target date was may 14, but from some reason it didn't happen.
I have a feeling that it will take some time.